### PR TITLE
PRODSEC-1257 Whitesource Transition to Snyk

### PR DESCRIPTION
**Pull request by sfc-gh-jfan**
Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

**Comment by sfc-gh-jfan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/snyk-pr.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: snyk-pr
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  whitesource:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'sfc-gh-snyk-sca-sa' }}
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:9]: This user should be added to the CLA exceptions

**Review by sfc-gh-mkeller — CHANGES_REQUESTED** [review]

File: .github/workflows/snyk-pr.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        fetch-depth: 0
+
+    - name: checkout action
+      uses: actions/checkout@v3
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.WHITESOURCE_ACTION_TOKEN }}
+        path: whitesource-actions
+
+    - name: PR
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:24]: Please be more descriptive with the name

File: .github/workflows/snyk-pr.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  whitesource:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'sfc-gh-snyk-sca-sa' }}
+    steps:
+    - name: checkout
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:11]: Please be consistent with these names, some of them are all lower case and some of them aren't

File: .github/workflows/snyk-pr.yml (hunk 1 above)
Comment (sfc-gh-jfan) [reply .github/workflows/snyk-pr.yml:9]: how can I add this user to CLA exceptions?

File: .github/workflows/snyk-pr.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        path: whitesource-actions
+
+    - name: Snyk Pull Request Scan Check
+      uses: ./whitesource-actions/snyk-pr
+      env:
+        pr_title: ${{ github.event.pull_request.title }}
+      with:
+        jira_token: ${{ secrets.jira_token_public_repo }}
+        gh_token: ${{ secrets.github_token }}
+        amend: false
```
Comment (sfc-gh-mkeller) [inline .github/workflows/snyk-pr.yml:31]:
```suggestion
        amend: false

```

**Review by sfc-gh-mkeller — APPROVED** [review]
:ship: :ship:

**Pull request merged**